minikube-maven-plugin/README.md

```diff
@@ -4,7 +4,7 @@ Minikube Maven Plugin
 ======================
 This plugin provides tasks and configuration to manage a minikube lifecycle for Maven developers.
 
-Now available on [Maven Central](https://mvnrepository.com/artifact/com.google.cloud.tools/minikube-maven-plugin).
+Now available on the [Central repository](https://mvnrepository.com/artifact/com.google.cloud.tools/minikube-maven-plugin).
```

Comment:
The problem is the URL, not the wording. mvnrepository.com is *NOT* Maven Central. You want maven.org

Reply:
Okay, changed the link to point to the latest Maven Central link (same as the badge).